apps/console/src/features/server-configurations/components/governance-connector-grid.tsx

```diff
@@ -0,0 +1,285 @@
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Typography from "@oxygen-ui/react/Typography";
+import { IdentifiableComponentInterface, LoadableComponentInterface } from "@wso2is/core/models";
+import { ContentLoader, GenericIcon } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, SyntheticEvent } from "react";
+import { AppConstants, EventPublisher } from "../../core";
+import "./governance-connector-grid.scss";
+import { getSettingsSectionIcons } from "../configs/ui";
+import { history } from "../../core";
+import { 
+    ArrowLoopRightUserIcon,
+    BuildingGearIcon, 
+    EnvelopeAtIcon, 
+    GearIcon, 
+    HexagonTwoIcon, 
+    PadlockAsteriskIcon, 
+    ShieldCheckIcon, 
+    ShieldUserPencilIcon, 
+    UserDatabaseIcon, 
+    UserDocumentIcon, 
+    UserGearIcon, 
+    UserPlusIcon 
+} from "@oxygen-ui/react-icons";
+import { ServerConfigurationsConstants } from "../constants/server-configurations-constants";
+import Card from "@oxygen-ui/react/Card";
+import { CardContent } from "@oxygen-ui/react";
+import { serverConfigurationConfig } from "apps/console/src/extensions";
+
+/**
+ * Props for the Governance connector configuration categories page.
+ */
+export interface GovernanceConnectorCategoriesGridInterface extends 
+    IdentifiableComponentInterface, LoadableComponentInterface {
+        /**
+         * Connector categories.
+         */
+        connectorCategories: any;
+        /**
+         * Dynamic connector catergories.
+         */
+        dynamicConnectors: any;
+    }
+
+/**
+ * Choose the application template from this page.
+ *
+ * @param props - Props injected to the component.
+ *
+ * @returns Application template select page.
+ */
+const GovernanceConnectorCategoriesGrid: FunctionComponent<GovernanceConnectorCategoriesGridInterface> = (
+    props: GovernanceConnectorCategoriesGridInterface
+): ReactElement => {
+    const { 
+        isLoading, 
+        ["data-componentid"]: componentId, 
+        connectorCategories,
+        dynamicConnectors 
+    } = props;
+
+    const eventPublisher: EventPublisher = EventPublisher.getInstance();
+
+    /**
+     * Handles template selection.
+     *
+     * @param e - Click event.
+     * @param id - Id of the template.
+     */
+    const handleConnectorSelection = (e: SyntheticEvent, route: string): void => {
+        history.push(route);
+    };
+
+    const handleDynamicConnectorSelection = (e: SyntheticEvent, connector: any): void => {
+        history.push(AppConstants.getPaths()
+            .get("GOVERNANCE_CONNECTOR_EDIT")
+            .replace(":categoryId", 
+                connector.categoryId)
+            .replace(":connectorId", connector.id));
+    };
+
+    if (isLoading) {
+        return <ContentLoader dimmer />;
+    }
+
+    const resolveConnectorCategoryIcon = (id : string): ReactElement => {
```

Comment:
Will it be possible to reuse connectorIcon object defined in apps/console/src/features/server-configurations/configs/ui.ts